tools/bin/ci_credentials.sh

```diff
@@ -1,26 +1,150 @@
 #!/usr/bin/env bash
 
 . tools/lib/lib.sh
+. tools/lib/gcp-token.sh
 
 set -e
 
+COMMAND_NAME=$1
+
+# all secrets will be loaded if the second argument is not present
+CONNECTOR_FULLNAME=${2:-all}
```

Comment:
should we move this after all the method declarations to cetnralize where all the logic for the `main` method exists?

Reply:
all bash function arguments have separate namespaces ($1, $2 ....) Thus script's arguments can be available in its "global" namespace only. It is a const variable and I defined it at the begging file.